packages/remix/test/integration/package.json

```
@@ -36,7 +36,7 @@
     "@sentry/types": "file:../../../types",
     "@sentry/utils": "file:../../../utils",
     "@vanilla-extract/css": "1.13.0",
-    "web-streams-polyfill": "3.2.1"
+    "@vanilla-extract/integrations": "6.2.4"
```

Need to fix this typo: 
```suggestion
    "@vanilla-extract/integration": "6.2.4"
```